diff --git a/node_modules/pdf-parse/index.js b/node_modules/pdf-parse/index.js
index e9fc367..b0f1dc3 100644
--- a/node_modules/pdf-parse/index.js
+++ b/node_modules/pdf-parse/index.js
@@ -3,7 +3,8 @@ const Pdf = require('./lib/pdf-parse.js');
 
 module.exports = Pdf;
 
-let isDebugMode = !module.parent; 
+//let isDebugMode = !module.parent; 
+let isDebugMode = false;
 
 //process.env.AUTO_KENT_DEBUG
 
diff --git a/node_modules/pdf-parse/lib/pdf.js/v1.10.100/build/pdf.js b/node_modules/pdf-parse/lib/pdf.js/v1.10.100/build/pdf.js
index d355798..3630f1a 100644
--- a/node_modules/pdf-parse/lib/pdf.js/v1.10.100/build/pdf.js
+++ b/node_modules/pdf-parse/lib/pdf.js/v1.10.100/build/pdf.js
@@ -6993,7 +6993,7 @@ var SVGGraphics = function SVGGraphics() {
         if (parseInt(process.versions.node) >= 8) {
           input = literals;
         } else {
-          input = new Buffer(literals);
+          input = new Buffer.from(literals);
         }
         var output = require('zlib').deflateSync(input, { level: 9 });
         return output instanceof Uint8Array ? output : new Uint8Array(output);
